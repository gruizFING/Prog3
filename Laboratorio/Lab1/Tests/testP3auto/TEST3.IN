TEST de EVAL
EVALUACION de constantes::
EVAL 0.25
EVAL 123
EVAL 3333.0
EVAL 66.99
EVAL X
VAR Y=12
EVAL X*Y+9
EVAL Y
EVAL X
EVAL (3*5)+X
EVAL (9-3)*5*5/3
EVAL (((9)*3)*3)
EVAL (((9)/3)/3)
EVAL X/0
EVAL (52/0)
EVAL expr
EVAL Y/*0
EVAL 89+-
VAR X=10
EVAL X+(3*2)
EVAL X+3*2
EVAL X*3+2
EVAL (X+3)*2
EVAL ((X+3)*2)
SALIR
