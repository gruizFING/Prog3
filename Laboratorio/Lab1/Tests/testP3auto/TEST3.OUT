Bienvenido a el mini-sistema:
> Comando no reconocido.
> Comando no reconocido.
> OK. 0.25
> OK. 123.00
> OK. 3333.00
> OK. 66.99
> ERROR: Falta definir variable o expresion.
> OK. Y agregada.
> ERROR: Falta definir variable o expresion.
> OK. 12.00
> ERROR: Falta definir variable o expresion.
> ERROR: Falta definir variable o expresion.
> OK. 50.00
> OK. 81.00
> OK. 1.00
> ERROR: Division por cero.
> ERROR: Division por cero.
> ERROR: Falta definir variable o expresion.
> ERROR: La expresion no es valida.
> ERROR: La expresion no es valida.
> OK. X agregada.
> OK. 16.00
> OK. 16.00
> OK. 32.00
> OK. 26.00
> OK. 26.00
> 